pkg/renew/renew_test.go

```diff
@@ -41,12 +41,18 @@ func TestValidate(t *testing.T) {
 			args:   []string{"abc"},
 			expErr: true,
 		},
+		"If there are arguments, as well as --all-namespaces, error": {
+			options: &Options{
+				AllNamespaces: true,
+			},
+			args:   []string{"abc"},
+			expErr: true,
+		},
```

Comment:
If arguments are specified:
- You would get an `Error from server (NotFound):` error with `--all-namespaces`, unless the first namespace that is [listed](https://github.com/cert-manager/cmctl/blob/c42c489dd0af74bb6a6568ea55e86cced39b4dc2/pkg/renew/renew.go#L154) happens to contain the certificate(s) specified in the arguments.
  - This behaviour doesn't sound ideal so I think it's more intuitive to disallow use of arguments with `--all-namespaces`; arguments should really only be used with `--namespace`.